core/tests/test_department_team.py

```diff
@@ -0,0 +1,40 @@
+# Third-Party Imports
+import pytest
+from django.db import transaction, utils
+
+# App Imports
+from core.models import Department, DepartmentalTeam
+from core.tests import CoreBaseTestCase
+
+
+class MyTestCase(CoreBaseTestCase):
+    def setUp(self):
+        self.new_department = Department(name="Operations")
+        self.new_department.save()
+
+    def tearDown(self):
+        self.new_department.delete()
+
+    def test_add_department_team(self):
+        count = DepartmentalTeam.objects.count()
+        new_team = DepartmentalTeam(
+            name="Accounts", department_id=self.new_department.id
+        )
+        new_team.save()
+        assert DepartmentalTeam.objects.count() == count + 1
+        new_team.delete()
+
+    def test_cannot_add_existing_team(self):
+        team = DepartmentalTeam(name="Accounts", department_id=self.new_department.id)
+        team.save()
+        with transaction.atomic():
```

Comment:
@kmazi the atomic block is actually necessary. In its absence, the following error is obtained:
```django.db.transaction.TransactionManagementError: An error occurred in the current transaction. You can't execute queries until the end of the 'atomic' block.```

Reply:
My point is that we don't need a db transaction for this test method especially because the TestCase base class handles transaction for all test methods. Therefore calling it explicitly is unnecessary. @fgkinus

Reply:
Noted. I have been able to remove the delete function calls that were eliciting the transaction errors.